### PR TITLE
Document kubeconfig property usage

### PR DESCRIPTION
**Pull request by manusa**
Relates to https://github.com/fabric8io/kubernetes-client/issues/1879

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ This will use settings from different sources in the following order of priority
 
 System properties are preferred over environment variables. The following system properties & environment variables can be used for configuration:
 
+* `kubeconfig` / `KUBECONFIG`: Name of the kubernetes config file to read (defaults to `~/.kube/config`)
```
Comment (rohanKanojia) [inline README.md]: Sorry, I'm not sure but isn't the property name `KUBERNETES_KUBECONFIG_FILE`? Have you tried configuring `kubeconfig` using this property?

Comment (manusa) [reply README.md]: Haven't tried it, but checked the code. This looks like a legacy property that doesn't match the other naming conventions:
https://github.com/fabric8io/kubernetes-client/blob/6a47d262e37bee71b9579551a9a56288761f02c2/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java#L110

So yes, the Java variable has `KUBERNETES_KUBECONFIG_FILE` this name, but not its  value neither the conversion to environment variables.

Comment (rohanKanojia) [reply README.md]: oh, yeah. You're right.

**Comment by manusa** [comment]
> Thanks a lot! Looks good with minor comment 👍
> 
> Thanks for adding a comment also to property. Could you please do the same for other properties also if you have time?

Yes, that's a good idea, will do.

Are these properties documented elsewhere (asciidoc, etc)?

**Comment by rohanKanojia** [comment]
I'm not sure whether they are documented anywhere :( . You can try looking into code and try getting hint via comments :-( .

**Comment by rohanKanojia** [comment]
[merge]

**Comment by rohanKanojia** [comment]
Thanks, I think it's a good start.

**Pull request merged**